src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -128,12 +132,11 @@ public boolean link() throws IOException, InterruptedException {
         if (clang == null) throw new IOException ("You specified an android ndk, but it doesn't contain "+ndk+"/toolchains/llvm/prebuilt/"+hostPlatformFolder+"/bin/clang");
         if (sdk == null) throw new IOException ("Can't find an Android SDK on your system. Set the environment property ANDROID_SDK");
 
-        boolean result = super.link();
-
-        if (!result) {
-            return false;
-        }
+        return super.link();
+    }
 
+    @Override
+    public boolean packageApp() throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'packageApp()' has an NPath complexity of 7680](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5152426)